Content.Client/Crayon/CrayonSystem.cs

```diff
@@ -64,4 +93,55 @@ protected override void FrameUpdate(FrameEventArgs args)
                 ("capacity",_parent.Capacity)));
         }
     }
+
+    private DecalPrototype? GetDecal(string decalId)
+    {
+        return decalId != null ?
+            _protoMan.Index<DecalPrototype>(decalId) : null;
+    }
```

Comment:
```suggestion
    private DecalPrototype? GetDecal(ProtoId<DecalPrototype>? decalId)
    {
        return decalId is { } id ? _protoMan.Index(id) : null;
    }
```
type safety is good to have here